docs/src/lib/transform.ts

```diff
@@ -43,7 +44,9 @@ function transformFunction({
 
   const methods = signaturesWithComments.map(transformSignature);
 
-  return { id, name, description, methods };
+  const source = sources?.[0]?.url;
+
+  return { id, name, description, methods, source };
```

Comment:
A better name would be `sourceUrl`, no?
```suggestion
  const sourceUrl = sources?.[0]?.url;

  return { id, name, description, methods, sourceUrl };
```